pkgs/development/tcl-modules/by-name/ji/jimlib/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitLab,
+  mkTclDerivation,
+}:
+
+mkTclDerivation rec {
+  pname = "jimlib";
+  version = "0.15.0-unstable-2024-10-31";
```

Comment:
Why do we need the unstable version?

Reply:
The git tag hasn't been updated in several years. This is the version in the source file, but there have been more commits since then that haven't bumped the version. Let me know if there's a more appropriate way to tag this

Reply:
> The git tag hasn't been updated in several years

0.16.0 is now out. Ask and you shall receive :)

Reply:
Updated and set rev to use the new tag